ckanext/harvester_dashboard/controllers.py

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+
+import logging
+from ckan.lib.base import BaseController, render
+from ckan.common import c, request, config, _
+from ckan.lib.base import render
+import ckan.plugins.toolkit as tk
+log = logging.getLogger(__name__)
+
+
+class HarvesterDashboardController(BaseController):
+    """Override the user controller to allow custom user search
```

Comment:
this doesn't seem to be overriding the user controller, only the base controller, or am I missing something?